docs/sources/next/javascript-api/k6-experimental/websockets/blob/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Blob'
+description: 'Interface that represents a blob, which is a file-like object of immutable, raw data; they can be read as text or binary data, or converted into a ReadableStream.'
+weight: 10
+---
+
+# Blob
+
+`Blob` is an interface that represents a blob, which is a file-like object of immutable, raw data; they can be read as text or binary data, or converted into a [ReadableStream](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/streams/readablestream).
+
+It's the type of the data received on [WebSocket.onmessage](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/websockets/websocket/websocket-onmessage) when `WebSocket.binaryType` is set to `"blob"`. 
+
+A `Blob` instance has the following methods/properties:
+
+<!-- vale off -->
+
+| Class/Property     | Description                                                                                                                                                                                 |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Blob.size          | The number of bytes of data contained within the `Blob`.                                                                                                                                     |
+| Blob.type          | A string containing the MIME type, or an empty string if the type could not be determined.                                                                                                  |
+| Blob.arrayBuffer() | Returns a `Promise` that resolves with the contents of the blob as binary data contained in an `ArrayBuffer`.                                                                               |
+| Blob.bytes()       | Returns a `Promise` that resolves with a `Uint8Array` containing the contents of the blob as an array of bytes.                                                                             |
+| Blob.slice()       | Returns a new `Blob` object which contains data from a subset of the blob on which it's called.                                                                                             |
+| Blob.stream()      | Returns a [ReadableStream](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/streams/readablestream) which upon reading returns the data contained within the `Blob`. |
+| Blob.text()        | Returns a `Promise` that resolves with a string containing the contents of the blob, interpreted as `UTF-8`.                                                                                |
```

Comment:
Do you (@mstoykov) think we need specific pages for these methods, as we have for WebSocket methods? 🤔 
I'm wondering whether specific examples of each method are really useful. Maybe just `slice()` which is the only one that can have multiple variants? Or just point to the MDN Web Docs?

Reply:
I think that we can do that in a future iteration. Maybe when Blob isn't in this module but globally available.